README.md

```diff
@@ -1,79 +1,134 @@
 # Hyperdash Python SDK
 
-The Hyperdash Python SDK is the official SDK for [Hyperdash.io](https://hyperdash.io). Once installed, the SDK automatically monitors your machine learning jobs.
+[Hyperdash](https://hyperdash.io) is a machine learning monitoring library, written in Python and capable of running alongside Tensorflow, Scikit-Learn, and other modelling libraries. Hyperdash provides visualizations similar to Tensorboard. It was developed with a focus on enabling fast knowledge gain.
 
-## Installation
+Use Hyperdash if you need model monitoring that:
 
-`pip install --upgrade pip && pip install hyperdash`
+* Is fast and easy to setup and use.
+* Tracks your hyperparmeters across different model experiments.
+* Graphs performance metrics (loss, reward, etc.) in real-time.
+* Saves a log file of your experiment job output.
+* Notifies you when a long-running experiment job has finished.
+* Can be viewed remotely via both a web and mobile app.
 
-## Usage
+Hyperdash is compatible with: **Python 2.7-3.6**
 
-The Hyperdash SDK requires a valid API key in order to function. Luckily, the `hyperdash login` (if you already have an account) and `hyperdash signup` (if you don't) commands will automatically install one for you.
+## Command Line Installation
+*Foreword: We care deeply about making Hyperdash fast and easy to install  on Linux, Mac, and Windows. If you find a snag along the way, please let us know at support@hyperdash.io!*
 
-If you'd rather manage your API key manually, then review the "API Key Storage" section below.
+Install Hyperdash from [pip](https://packaging.python.org/tutorials/installing-packages/#requirements-for-installing-packages):
+```bash
+$ pip install --upgrade pip && pip install hyperdash
+```
+Installing within a python virtual environment such as [virtualenv](https://github.com/pypa/virtualenv) or [conda](https://github.com/conda/conda) is recommended.
+```bash
+# Login if you have an account
+$ hyperdash login
 
-### Standalone Script
-The easiest way to use the Hyperdash SDK is to simply prefix any terminal command with `hyperdash run`:
+# Or signup free with an email
+$ hyperdash signup
+```
+After `login` or `signup`, an API key is saved to your local machine for automatic authorization. If you'd rather manage your API key manually, then review the "API Key Storage" section below.
 
+You're ready to use Hyperdash! Make sure Hyperdash works by running:
 ```
-hyperdash run -n "My test python script" python my_test_script.py
+$ hyperdash demo
 ```
 
-or
+## Learn Hyperdash in 30 seconds
```

Comment:
I think this intro needs a little work. We have three ways of using Hyperdash (not including Jupyter) and we should probably introduce them in order of increasing complexity so we don't lose users in the funnel.

I.E I would start off by saying, if you just want to know when your job is complete and view your logs remotely, use the hyperdash run command blah blah blah

Alternatively, you can decorate the main function in your python script with the monitor decorator blah blah blah

Finally, we have an experiment API which blah blah blah

Reply:
Basically get it working with their code as fast as possible, then teach them how to do more complicated stuff once they've seen how easy it is to get the basics